codalab/apps/web/static/less/worksheets.less

```diff
@@ -54,6 +54,9 @@
          top:-7px;
       }
    }
+   > .container {
+     width:auto;
```

Comment:
Another example of where the formatting is all off and hard to read. 
![screen shot 2015-04-29 at 1 32 42 pm](https://cloud.githubusercontent.com/assets/123099/7400797/4584cdba-ee74-11e4-87c1-dc69b702d210.png)